samples/graph-teams-tag/nodejs/README.md

```diff
@@ -0,0 +1,123 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates how to use CRUD Graph operations related to team tags.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+contentType: samples
+createdDate: "02-08-2022 00:02:15"
+---
+
+# This is a sample application shows the usage of Graph CRUD operations related to team tags.
```

Comment:
This is a sample application that shows the usage of Graph CRUD operations related to team tags.